desktop/cmp/chart/theme/Dark.js

```diff
@@ -64,6 +64,7 @@ export const DarkTheme = {
         lineColor: '#707073',
         minorGridLineColor: '#505053',
         tickColor: '#707073',
+        // default tickWidth here ??
```

Comment:
Could see doing this to ensure consistant behavior in client apps, but could also make the argument that we should embrace the library's decision regarding this default, and make it the app's responsibility to adjust if necessary